### PR TITLE
Bug 1183870 - Set audio channel as none when audio channel is inactive

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4cad004bb986369d6a77f5a62fb12d935879168f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=be3ce1d608274d79ad51998dbfc758c43549cbd3

File: apps/system/js/audio_channel_service.js [hunk 1]
```diff
@@ -122,10 +122,11 @@
         if (!isBackground) {
           this.publish('visibleaudiochannelchanged', channel);
         }
-        this.publish('audiochannelchangedasactive', channel);
+        this.publish('audiochannelchanged', channel);
       } else {
         this._resetAudioChannel(audioChannel);
         this._resumeAudioChannels();
+        this.publish('audiochannelchanged', { channel: 'none' });
```
Comment (dominickuo) [inline apps/system/js/audio_channel_service.js]: @evanxd This line should be the fix of the root cause, right? for the others I think we just change the event names from `audiochannelchangedasactive` to  `audiochannelchanged`.

Comment (evanxd) [inline apps/system/js/audio_channel_service.js]: @dominickuo , Yes.

**Pull request merged**